### ISSUE TITLE
New layout? T060138X

### ISSUE THREAD

**Issue by flopp999** [johanmeijer/grott#470]
- Grott automatic protocol detection
- Grott data record length 413
- layout   :  T060138X
- no matching record layout found, try generic
- Record layout used :  T060138X
- Growatt data decrypted V2
- Grott Growatt data decrypted

but I do not find 138X as an layout

after the plain data it says

- Grott data ack record or data record not defined no processing done

**Comment by johanmeijer** [comment]
I do not think this is an important record (not a real datarecord). It is ok to ignore. I will change the log messages in a next release.
